### DISCUSSION TITLE
Build Adonisjs Inertia to production

### DISCUSSION THREAD

**Discussion by timesignature** [adonisjs/core#4468] [Help] [answered]
Hi Guys, how do you build adonisjs inertia project for production. i tried using node ace build but its giving me the following error. Failed to load url inertia/app/ssr.tsx (resolved id: inertia/app/ssr.tsx). Does the file exist?

thanks in advance

**Comment by Julien-R44** [comment]
Please set NODE_ENV=production when running your application in production

**Reply by timesignature** [reply]
@Julien-R44 thanks for your response brother but im getting the following on the browser. Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'

**Comment by timesignature** [comment, marked as answer]
I was able to solve this one on my own. after building your adonisjs project, you have to move the following files and folders into the build folder

- inertia
- .env
- tailwind.config.js
- tmp

**Reply by IzliaB** [reply]
Hello! I'm having issues with the styles when dockerizing the project. I configured the styles with tailwindcss and vite. I'm using edge as a template in Adonis, but the styles are not working properly in the container. I tried your solution but it didn't work, I really don't know what else to try. Could you take a look at my Dockerfile? Believe me, I've been trying to dockerize and deploy the project for two weeks now, and I'm about to give up on Adonis and go back to Node because I really don't know why I'm having trouble with the styles. I mean, I use playwright to generate PDF files, but playwright only makes everything else processed by adonis with vite, tailwindcss, and edge.

**Reply by timesignature** [reply]
@IzliaB, sorry mate i have not worked with edge template before. i only used inertia or writing rest api
